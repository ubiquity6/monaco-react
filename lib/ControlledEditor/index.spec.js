"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _ = _interopRequireDefault(require("."));

var _react2 = require("@testing-library/react");

require("@testing-library/jest-dom/extend-expect");

describe('<ControlledEditor />', function () {
  it('should check render with snapshot', function () {
    var component = (0, _react2.render)( /*#__PURE__*/_react["default"].createElement(_["default"], null));
    expect(component).toMatchSnapshot();
  });
});