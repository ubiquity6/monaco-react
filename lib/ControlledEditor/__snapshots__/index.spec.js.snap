// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ControlledEditor /> should check render with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        style="display: flex; position: relative; text-align: initial; width: 100%; height: 100%;"
      >
        <div
          style="display: flex; height: 100%; width: 100%; justify-content: center; align-items: center;"
        >
          Loading...
        </div>
        <div
          style="width: 100%; display: none;"
        />
      </section>
    </div>
    <script
      src="https://cdn.jsdelivr.net/npm/monaco-editor@0.20.0/min/vs/loader.js"
    />
  </body>,
  "container": <div>
    <section
      style="display: flex; position: relative; text-align: initial; width: 100%; height: 100%;"
    >
      <div
        style="display: flex; height: 100%; width: 100%; justify-content: center; align-items: center;"
      >
        Loading...
      </div>
      <div
        style="width: 100%; display: none;"
      />
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
